lib/business_time/time_extensions.rb

```diff
@@ -2,27 +2,20 @@
 module BusinessTime
   module TimeExtensions
     include BusinessTime::Currency
+
     # True if this time is on a workday (between 00:00:00 and 23:59:59), even if
     # this time falls outside of normal business hours.
     def workday?(*currency)
-      currency = args(*currency)
+      currencies = args(*currency)
+      # This may BREAK currencies where weekday are not standard (IE Arabic). In that
+      # case those currencies will have 3 non work days per week
       return false unless weekday?
```

Comment:
What is the reason of setting a local variable before early return? Swap please L#12 and L#9.